ui-v2/app/templates/dc/services/show/upstreams.hbs

```diff
@@ -26,8 +26,9 @@
   {{#let (sort-by (comparator 'service' sort) filtered) as |sorted|}}
       <ChangeableSet @dispatcher={{searchable 'service' sorted}} @terms={{search}}>
         <BlockSlot @name="set" as |searched|>
-          <ConsulServiceList
+          <ConsulUpstreamList
```

Comment:
Yoikes! Some copypasta seems to have completely replaced the UpstreamList with the plain service one, this brings it back.